fleet_adapter_mir_actions/fleet_adapter_mir_actions/rmf_cart_detection.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+from urllib.error import HTTPError
+
+
+class CartDetection:
```

Comment:
Seems like this should be `ABC` and all the methods should have `@abstractmethod`.

Reply:
This makes sense, thanks for pointing it out.